mbedtls-sys/vendor/library/bignum.c

```diff
@@ -1550,7 +1550,7 @@ __attribute__ ((noinline))
 #endif
 void mpi_mul_hlp( size_t i, mbedtls_mpi_uint *s, mbedtls_mpi_uint *d, mbedtls_mpi_uint b )
 {
-    mbedtls_mpi_uint c = 0;
+    mbedtls_mpi_uint c = 0, t = 0;
```

Comment:
this was removed in https://github.com/mobilecoinfoundation/rust-mbedtls/pull/106 but is needed for i686 builds, https://github.com/mobilecoinfoundation/rust-mbedtls/blob/e966091845f27d49b0d9ba91fc99125e0c5f111c/mbedtls-sys/vendor/include/mbedtls/bn_mul.h#L166